Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/outbound/StandardOutboundDocQueryTest.java

```diff
@@ -85,7 +88,7 @@ public void testrespondingGatewayCrossGatewayQueryforNullEndPoint() throws Excep
         NhinTargetSystemType target = null;
         final DocQueryAuditLogger auditLogger = getAuditLogger(true);
         AssertionType assertion = new AssertionType();
-
+        
         StandardOutboundDocQuery entitydocqueryimpl = new StandardOutboundDocQuery(strategy, service, policyChecker) {
```

Comment:
Instead of creating a new class definition and copy/pasting the same code 5 times in this file, perhaps a helper method to give you this object could be used? Smaller file size, less number of lines, and easier to maintain in the future.